src/components/PersonaCard/PersonaCard.less

```diff
@@ -168,23 +168,33 @@
   overflow-y: auto;
 }
 
-.ms-PersonaCard-detailChat { margin-left: 0 !important; }
```

Comment:
Removing all these !importants makes me a little nervous, but I tested them, and didn't see any regressions!